src/Components/Modal.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import styled from 'styled-components';
+
+const Modal = ({ children, top, right, bottom, left }) => {
+  return createPortal(
+    <StyledModal top={top} right={right} bottom={bottom} left={left}>

```

Comment:
```suggestion
    <StyledModal {...{top,bottom,right,left}}>
```

props는 객체형식으로 전달되고, 객체 단축속성명을 이용해서 위와 같이 간결하게 표현할수도 있습니다!
동작상의 차이는 없고, 가독성적인 측면도 관점에 따라 견해가 다를 수 있지만, 이런 방법이 있다고 알아두시면 됩니다!

Reply:
관련하여 아랫쪽에 질문 올렸습니다!